.github/workflows/build-test-ci.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'pnpm'
+    - name: Enabling pre-post scripts
+      run: echo 'enable-pre-post-scripts=true' > ~/.npmrc
```

Comment:
Thanks @atifcppprogrammer , would `pnpm config set enable-pre-post-scripts true` work instead?  Just thinking overwriting the whole `.npmrc` file could cause unexpected behaviour in future.

Reply:
@coel  Good call ! let me patch that up real quick !